ext/intl/tests/timezone_getIanaID.phpt

```diff
@@ -0,0 +1,23 @@
+--TEST--
+IntlTimeZone::getIanaID
+--EXTENSIONS--
+intl
+--SKIPIF--
+<?php if (version_compare(INTL_ICU_VERSION, '74.0') < 0) die('skip for ICU < 74.0'); ?>
```

Comment:
nit: the relational operators are the other way around what looks logical based on only the returned message.

```suggestion
<?php if (version_compare(INTL_ICU_VERSION, '74.0') < 0) die('skip for ICU >= 74.0'); ?>
```